src/parser/parser.py

```diff
@@ -11,10 +11,11 @@
 class Parser:
     def __init__(self, language='english'):
         # constants
-        self.DATA_DIR = './decompiler/decompiled-data/'
+        self.DATA_DIR = os.path.join(os.getenv('OUTPUT_DIR',"./decompiler"),'decompiled-data/')
```

Comment:
Related to my comment on `.env.example`, this `DATA_DIR` is specifically decompiled-data. And `OUTPUT_DIR` is the data that will be used externally for analysis and on the wiki. So this should not be based on `OUTPUT_DIR`

Reply:
I think I've updated all the locations to use the `WORK_DIR` var, and in the parsers the constant that uses the `OUTPUT_DIR` var